### PR TITLE
Remove PCIUAddress from Betamocks setup

### PR DESCRIPTION
**Pull request by rjohnson2011**
## Summary

Removes EVSS::PCIUAddress from Betamocks setup as part of EVSS PCIU migration epic.

## Related issue(s)

[Github Ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/79044)

## Testing done

- Passes all specs. 

## What areas of the site does it impact?
Betamocks setup cleanup.

### CODE REVIEW

File: docs/setup/betamocks.md [hunk 1]
```diff
@@ -58,16 +58,6 @@ Each service description has a `base_uri` (pulled from Settings)
 - `method:` a symbol of the http verb `:get`, `:post`, `:put`...
 - `path:` the path that combined with the base_uri makes a full URI
 - `file_path:` where to save the file (relative to betamocks' cache dir)
-```yaml
```
Comment (RachalCassity) [inline docs/setup/betamocks.md:61]: Not sure if lines 61 and 62 need to be removed

File: docs/setup/betamocks.md [hunk 2]
```diff
@@ -58,16 +58,6 @@ Each service description has a `base_uri` (pulled from Settings)
...
-:services:
-
-# EVSS::PCIUAddress
```
Comment (RachalCassity) [inline docs/setup/betamocks.md:64]: You might need to replace this with Contact Information so other devs can use betamocks

**Review by RachalCassity — CHANGES_REQUESTED** [review]

File: docs/setup/betamocks.md (hunk 2 above)
Comment (rjohnson2011) [reply docs/setup/betamocks.md:64]: Sounds good, I'll make this update.

File: docs/setup/betamocks.md (hunk 1 above)
Comment (rjohnson2011) [reply docs/setup/betamocks.md:61]: Reinstated with a new example. Also linked to settings config file above.

File: docs/setup/betamocks.md (hunk 2 above)
Comment (rjohnson2011) [reply docs/setup/betamocks.md:64]: How's this with an updated example?

File: docs/setup/betamocks.md [hunk 3]
```diff
@@ -61,12 +61,15 @@ Each service description has a `base_uri` (pulled from Settings)
...
 
-# EVSS::PCIUAddress
-- :base_uri: <%= URI(Settings.evss.url).host %>
+# VA Profile / Vet360
+  :base_uri: <%= "#{URI(Settings.vet360.url).host}:#{URI(Settings.vet360.url).port}" %>
   :endpoints:
-  - :method: :get
-    :path: "/wss-pciu-services-web/rest/pciuServices/v1/states"
-    :file_path: "evss/pciu_address"
+    - :method: :get
```
Comment (RachalCassity) [inline docs/setup/betamocks.md:67]: You can remove the demographics

**Review by RachalCassity — CHANGES_REQUESTED** [review]

File: docs/setup/betamocks.md (hunk 2 above)
Comment (RachalCassity) [reply docs/setup/betamocks.md:64]: Hmm... I guess you can delete it. It's only for demographics.

Comment (RachalCassity) [reply docs/setup/betamocks.md:64]: Sorry for the back and forth. Just remove it

Comment (rjohnson2011) [reply docs/setup/betamocks.md:64]: Updated!

File: docs/setup/betamocks.md [hunk 4]
```diff
@@ -61,12 +61,12 @@ Each service description has a `base_uri` (pulled from Settings)
...
-- :base_uri: <%= URI(Settings.evss.url).host %>
+# VA Profile / Vet360
+  :base_uri: <%= "#{URI(Settings.vet360.url).host}:#{URI(Settings.vet360.url).port}" %>
   :endpoints:
-  - :method: :get
-    :path: "/wss-pciu-services-web/rest/pciuServices/v1/states"
-    :file_path: "evss/pciu_address"
+  - :method: :post
+    :path: "/profile-service/profile/v3/*/*"
+    :file_path: "vet360/profile-service/default"
```
Comment (RachalCassity) [inline docs/setup/betamocks.md:69]: Where are these file paths from? Just curiou

**Review by RachalCassity — APPROVED** [review]

**Pull request merged**